src/test/kotlin/no/nav/pensjon/simulator/tpregisteret/TpregisteretClientTest.kt

```diff
@@ -0,0 +1,115 @@
+package no.nav.pensjon.simulator.tpregisteret
+
+import no.nav.pensjon.simulator.tech.trace.TraceAid
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
```

Comment:
Forslag: Bruk `Kotest` istedenfor JUnit (slik det er gjort i [PenAlderspensjonClientTest](https://github.com/navikt/pensjonssimulator/blob/main/src/test/kotlin/no/nav/pensjon/simulator/alderspensjon/client/pen/PenAlderspensjonClientTest.kt)).
Fordelene er beskrevet i [JUnit 5 vs. Kotest: A Comparison](https://www.baeldung.com/kotlin/kotest-vs-junit-5):

> It leverages Kotlin’s language features and provides a concise and expressive syntax for writing tests.

Reply:
takk for forslaget og lenker 👍